src/components/react/SgidIndexSearchCard.tsx

```diff
@@ -0,0 +1,101 @@
+import startCase from 'lodash.capitalize';
+
+import { convertProductType, getArcGisHubUrl, getFeatureServiceUrl } from '@utils/data';
+
+import ExternalLink from '@components/react/ExternalLink';
+import { type StewardshipRecord } from '@models/products/sgid/types';
+import { Fragment } from 'react/jsx-runtime';
+import dnrLogo from './dnr-logo.svg';
+import udotLogo from './udot-logo.svg';
+import ugrcLogo from './ugrc-logo.svg';
+
+interface Props {
+  astroSite: URL | undefined;
+  record: StewardshipRecord;
+  resultIndex: number;
+}
+export default function SgidIndexSearchCard({ astroSite, record, resultIndex }: Props) {
+  type appLink = {
+    url: string;
+    title: string;
+  };
+  const apps = [] as appLink[];
+  if (record.inActionUrl) {
+    apps.push({ url: record.inActionUrl, title: 'Data in action' });
+  }
+  if (record.server?.host) {
+    apps.push({ url: getFeatureServiceUrl(record.server), title: 'Feature service' });
+  }
+  if (record.hub?.itemId) {
+    apps.push({ url: getArcGisHubUrl(record.hub), title: 'ArcGIS Hub item' });
+  }
+
+  let urlString = record.productPagePath;
+  let logo = ugrcLogo.src;
+  if (urlString.startsWith('https://')) {
+    const url = new URL(urlString);
+    switch (url.hostname) {
+      case 'dwr-data-utahdnr.hub.arcgis.com':
+        logo = dnrLogo.src;
+        break;
+      case 'data-uplan.opendata.arcgis.com':
+        logo = udotLogo.src;
+        break;
+    }
+  } else {
+    urlString = (import.meta.env.DEV ? 'http://localhost:4321' : astroSite?.href) + urlString;
+  }
+
+  return (
+    <div className="flex grow flex-col">
+      {record.productPagePath.length > 0 ? (
```

Comment:
**though**: what do you think about if `record.productPagePath.length > 0`, we then choose the hub, server, or inaction url to use instead (in that order).

<img width="351" alt="image" src="https://github.com/agrc/gis.utah.gov/assets/325813/005acad6-816d-4751-bf88-634b5ec13011">